code/modules/reagents/reagent_containers.dm

```diff
@@ -11,6 +11,8 @@
 	var/spawned_disease = null
 	var/disease_amount = 20
 	var/spillable = FALSE
+	var/pH_immunne = TRUE//true for now, just so things that shouldn't melt don't.

```

Comment:
immune is misspelled
also make the immune stuff a bitflag (ask if you need to know what bitflags/bitfields are) (it'd appreciate it if the other stuff was put into the bitfield but honestly don't do it unless you want to as i don't mind either way seeing that this pr took too long already)

Reply:
I want to do my best. I tried doing what you said, from reading off the tgwiki on it. Please re-review my edit.

Reply:
I think I did it. Though I'm not sure I did it right. Please check this.

Reply:
It doesn't compile at the moment, I don't think I've done it right, I've DM'd you to try to figure it out.

Reply:
pH flags work, temp flags partially work.

Reply:
Seems to work.